docs/Readme.md

```diff
@@ -51,6 +51,8 @@ you.
 
 * [Training on the Cloud with Amazon Web Services](Training-on-Amazon-Web-Service.md)
 * [Training on the Cloud with Microsoft Azure](Training-on-Microsoft-Azure.md)
+* [Using Docker](Using-Docker.md)
```

Comment:
You need to rearrange this section - "Cloud Training" isn't necessarily applicable anymore.

Reply:
Moved